### DISCUSSION TITLE
How to add a folder that is not tracked in the git to a deployment connected to github

### DISCUSSION THREAD

**Discussion by shriharip** [vercel/community#7419] [Help] [answered]
### Summary

I have a folder that contains some certificates and` .pk12` and ` .pem` files. I do not have it tracked in git for obvious reasons.

I have a vercel deployment that is connected to my GitHub and deploys on push to `dev` branch.
Now how can I upload only this folder to the deployment. 
I know how to use `.env` and set it up, however this is a folder with a different file format.

PS: It is a nextjs project.

### Steps to Reproduce

1. Create a folder with secret files
2. add the folder to `.gitignore`
3. Connect the git repo to vercel and set up deployment.
4. upload this folder.

**Comment by pawlean** [comment]
Hi @shriharip!

My understanding is that you want a folder to be included in the deployment to Vercel without being tracked by Git, is that correct?

Maybe you could use external storage (e.g. S3) to store the files, and download them during deployment?

**Comment by shriharip** [comment]
Hello @pawlean ,

Yes, you are absolutely right. 
Do you think that is the only way to achieve it ? Thanks

**Reply by pawlean** [reply, marked as answer]
@shriharip It's the route I would investigate, as this unfortunately isn't built-in to the product (as of today!)

[a comment by an automated account is not shown]
